### ISSUE TITLE
[FEATURE-REQUEST] Full (out of the box) support of react-redux just like Nuxt.js with its Vuex Store

### ISSUE THREAD

**Issue by handhikadj** [zeit/next.js#10151]
Implementing react-redux, moreover with redux-thunk into Next.js is really pain in the ass. Huge obstacle along the journey. Please make my dream comes true, to say the least

**Comment by rafaelalmeidatk** [comment]
Redux is not necessary on all projects, so this is not going to happen. Moreover, once #9133 is ready, I imagine it will be simpler to add redux to your projects.

**Comment by handhikadj** [comment]
@rafaelalmeidatk sorry if this one is out of context. however, do you have any working example of nextjs with redux and redux thunk?

**Comment by rafaelalmeidatk** [comment]
Yes, we have a lot of examples here: https://github.com/zeit/next.js/tree/canary/examples
The redux example: https://github.com/zeit/next.js/tree/canary/examples/with-redux
Redux with redux-thunk: https://github.com/zeit/next.js/tree/canary/examples/with-redux-thunk

**Comment by timneutkens** [comment]
At this point we're not planning to add redux to Next.js as there's many state management solutions and they all have different trade-offs. We do have examples for most state management solutions in the examples directory like @rafaelalmeidatk said 👍 https://github.com/zeit/next.js/tree/canary/examples

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
